Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -455,10 +455,17 @@ public class Event: NosManagedObject, VerifiableEvent {
         return fetchRequest
     }
     
-    @nonobjc public class func event(by replaceableID: RawReplaceableID, author: Author) -> NSFetchRequest<Event> {
+    @nonobjc public class func event(
+        by replaceableID: RawReplaceableID, 
+        author: Author,
+        kind: Int64
```

Comment:
I suppose this can't be `EventKind` because we might see a kind we don't have a case for in an naddr?